index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Mobile Version</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <div class="header-welcome">
+      <a href="/">
+        <img class="some-image" src="images/Welcome.svg" alt="Welcome" />
+      </a>
+    </div>
+
+    <nav>
+      <div class="menu-container">
+        <img class="some-image" src="images/icon-menu.svg" alt="Menu Icon">
+      </div>
+    </nav>
+  </header>
+
+  <main class="header-top">
+    <section class="headline">
+      <div class="container-header">
+        <h1>Hey there. I'm Tofunmi</h1>
+        <h2>I'm a software developer</h2>
+      </div>
```

Comment:
- [ ] [OPTIONAL] Can you kindly reduce the margin-top between the header and headline sections? for now, it seems to be so big compared to the original design in Figma, see the difference below 👇 

- Your design
![image](https://user-images.githubusercontent.com/55465133/133653403-49bb5314-7811-420f-861e-b3e62452dc94.png)

- Figma design
![image](https://user-images.githubusercontent.com/55465133/133653454-52b0a211-edf8-470a-bcf1-ff9c3d262257.png)

Can you see the difference? it can be great to fix this 😉